### ISSUE TITLE
Combined search in combo box for customers and projects

### ISSUE THREAD

**Issue by jaragunde** [Igalia/phpreport#271]
When adding a task, the customer field is usually just used as a filter for the project field. It would be ideal we had only one combo box where users could type project or customer names to get a filtered list of projects.

It could look like this:

```

--------------------------------
| Goog                      |\/|
--------------------------------
 |  Chromium project - Google  |
 |  Hangouts project - Google  |
 -------------------------------
 
--------------------------------
| Ch                        |\/|
--------------------------------
 |  Chromium project - Google  |
 |  Chakra project - Microsoft |
 -------------------------------
```

**Comment by jaragunde** [comment]
I found this example in Sencha forums, it should work for our purpose: http://extjs.com/forum/showthread.php?p=209279#post209279

**Comment by jaragunde** [comment]
Fixed in the same patch set as #248:
```
```
65c0490 (HEAD -> master, origin/master, origin/HEAD) [#248] [#271] Remove "client" as a compulsory field for tasks [GitHub]
d16c4fa (tony/remove-client) [#248] Remove customer relations completely from tasks and userTasksReports [Tony Thomas]
03d9055 [#248] Replace Project-Customer report in viewUserDetails with Project-Worked hours [Tony Thomas]
849028d [#248] Do not populate task/template table with customerid anymore [Tony Thomas]
a28cf41 [#248][Cleanup] Remove use of RequestDAO from codebase [Tony Thomas]
bdb2a60 [#248] Remove customer-project relations from viewProjects.php [Tony Thomas]
ed2e8d9 [#248] Allow filter via projectname-customer name in userTasksReport.php [Tony Thomas]
e3ecb60 [#248] Show only active projects on taks page and the full list on viewPorjects [Tony Thomas]
36b04ef [#248] Remove the project/customer report in the projects summary page [Tony Thomas]
e5f8866 [#248] Replace user-customer grid with total hours for all users for a project [Tony Thomas]
34beaa5 [#248] Allow filter via projectname-customer name in projectDetails.php [Tony Thomas]
6bb6325 [#248] Disable customercombobox, make it show up below project combo [Tony Thomas]
771ae87 [#271] Populate the task screen and database with the correct customer details [Tony Thomas]
cf5a5c2 [#271] Show projects without a client assigned too in the tasks screen [Tony Thomas]
3855718 [#271] Show up customers along with projects in tasks screen [Tony Thomas]
3147d3d [#248] Migration script to populate customerId in projects table [Tony Thomas]
89622d5 [#248] Initial implementation, add customer field to project management screen [Tony Thomas]
